cmd/query/app/querysvc/query_service.go

```diff
@@ -44,6 +44,7 @@ type QueryServiceOptions struct {
 // StorageCapabilities is a feature flag for query service
 type StorageCapabilities struct {
 	ArchiveStorage bool `json:"archiveStorage"`
+	// TODO: Maybe add metrics Storage here
```

Comment:
can you elaborate?

Reply:
Well apparently query service has a functionality to let us know what kind of features are supported by the query service `GetCapabilities()`. So I thought It would be a good idea to add metrics storage to this list as this is an optional feature-similar to archive storage

Reply:
Will include this change in a separate PR